Scenes/DetailScreen/Controller/MovieDetailViewController.swift

```diff
@@ -0,0 +1,189 @@
+extension MovieDetailViewController {
+    
+    func updateUI() {
+        if let backdropImage = viewModel.movieDetail?.backgroundImage {
+            backGroundImageView.getImageFromUrl(url: backdropImage)
+        }
+        if let posterImage = viewModel.movieDetail?.posterImage {
+            avatarImageView.getImageFromUrl(url: posterImage)
+        }
+        nameMovieLabel.text = viewModel.movieDetail?.title
+        overviewLabel.text = viewModel.movieDetail?.overview
+        dateLabel.text = viewModel.movieDetail?.releaseDate
+        if let averageRating = viewModel.movieDetail?.voteAverage {
+            averageRatingLabel.text = "\(averageRating)"
+        }
+        if let rateCount = viewModel.movieDetail?.rateCount {
+            rateCountLabel.text = "\(rateCount)"
+        }
+        if let popularity = viewModel.movieDetail?.popularity {
+            popularityLabel.text = "\(popularity)"
+        }
+    }
+    
+    func fetchMovieDetail() {
+        viewModel.fetchMovieDetails()
+        viewModel.fetchReviewOfMovie()
+    }
+}
+//MARK: Configure view
+extension MovieDetailViewController {
+    private func config() {
+        configViewModel()
+        configView()
+        configTableView()
+        configImageView()
+        configCollectionView()
+    }
+    private func configCollectionView() {
+        categoryCollectionView.delegate = self
+        categoryCollectionView.dataSource = self
+        categoryCollectionView.registerNib(cellName: CategoryCollectionViewCell.className)
+    }
+    
+    func configViewModel() {
+        viewModel.receivedNewData = { [weak self] in
+            DispatchQueue.main.async {
+                self?.updateUI()
+            }
+        }
+    }
+    
+    private func configImageView() {
+        avatarImageView.layer.cornerRadius = Defines.cornerRadius
+        backGroundImageView.layer.cornerRadius = Defines.cornerRadius
+    }
+    
+    private func configView() {
+        subViewAboutMovie.isHidden = false
+        aboutMovieView.isHidden = false
+        reviewsView.isHidden = true
+        subViewReviews.isHidden = true
+    }
+    
+    private func configTableView() {
+        reviewsTableView.dataSource = self
+        reviewsTableView.delegate = self
+        reviewsTableView.registerNib(cellName: ReviewsTableViewCell.className)
+    }
+}
+// MARK : TableView DataSource
+extension MovieDetailViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView,
+                   numberOfRowsInSection section: Int) -> Int {
+        return viewModel.getNumberOfRows(section)
+    }
+    
+    func tableView(_ tableView: UITableView,
+                   cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = reviewsTableView.dequeueReusableCell(withClass: ReviewsTableViewCell.self, for: indexPath)
+        if (viewModel.movieReview?.results.indices)! ~= indexPath.row {
+            let review = viewModel.getReviewMovieOfCell(indexPath)
```

Comment:
Why did you force unwrap the optional value instead of using optional binding (if let) ?